packages/jokul/MIGRATION.md

```diff
@@ -0,0 +1,165 @@
+```
+
+og i tillegg må man nå også spesifikt importere stilene til lenker side de heller ikke
+lenger er en del av core
+
+```diff
++ @use "@fremtind/jokul/styles/components/link";
+```
+
+Dersom man bruker ScreenReaderOnly må denne importen oppdateres
+
+```diff
+- import { ScreenReaderOnly } from "@fremtind/jkl-core";
++ import { ScreenReaderOnly } from "@fremtind/jokul";
+```
+
+Dersom man bruker TS hjelpeklasser som feks `WithChildren`
+
+```diff
+- import type { WithChildren } from "@fremtind/jkl-core";
++ import type { WithChildren } from "@fremtind/jokul";
+```
+
+## Fonter
+
+Tidligere lå font-filer tilgjengelig på stien
+`node_modules/@fremtind/jkl-webfonts/fonts/`
+mens de nå er flyttet til
+`node_modules/@fremtind/jokul/src/fonts/`
+
+Dersom du har script som kopierer font-filer må disse oppdateres med ny path
+
+SCSS for å dra inn stilene på oppdateres
+
+```diff
+- @use "@fremtind/jkl-webfonts/webfonts";
++ @use "@fremtind/jokul/styles/fonts/webfonts";
+```
+
+Hvis du sender inn `$webfonts-dir` variabelen er det også her path må oppdateres.
+
+## Komponenter
+
+Med noen unntak gjøres dette på samme måte for alle
+
+### Autosuggest
+
+Komponenten er flyttet ut fra TextInput
+
+import { Autosuggest } from "@fremtind/jkl-text-input-react";
+
+Import av React komponent endres
+
+```diff
+- import { Autosuggest } from "@fremtind/jkl-text-input-react";
++ import { Autosuggest } from "@fremtind/jokul/components/autosuggest";
+```
+
+Import av styling
+
+```diff
++ @use "@fremtind/jokul/styles/components/autosuggest";
+```
+
+### ExpandButton
+
+ExpandButton har blitt renamet til Expander
+
+Import av React komponent endres
+
+```diff
+- import { ExpandButton } from "@fremtind/jkl-expand-button-react";
++ import { Expander } from "@fremtind/jokul/components/expander";
+```
+
+Import av styling endres
+
+```diff
+- @use "@fremtind/jkl-expand-button/expand-button";
++ @use "@fremtind/jokul/styles/components/expander";
+```
+
+Prop `density` er fjernet, her kan man heller bruke `data-density` HTML attributet.
+
+Prop `hideLabel` er fjernet, ønskes en skjult label kan `aria-label` brukes i
+stedet for `children`.
+
+Komponenten har blitt polymorf og dermed fått en `as` prop. Typing
+av `onClick` on `onKeyDown` vil da være avhengig av hva som sendes inn her.
+
+### Icons
+
+Icons har blitt entallifisert og heter nå Icon
+
+Import av React komponent endres
+
+```diff
+- import { CloseIcon } from "@fremtind/jkl-icons-react";
++ import { CloseIcon } from "@fremtind/jokul/components/icon";
+```
+
+Import av styling endres
+
+```diff
+- @use "@fremtind/jkl-icons/icons";
++ @use "@fremtind/jokul/styles/components/icon";
+```
+
+### Røkla
+
+Import av React komponent endres
+
+```diff
+- import { Feedback } from "@fremtind/jkl-feedback-react";
++ import { Feedback } from "@fremtind/jokul/components/feedback";
+```
+
+Import av styling endres
+
+```diff
+- @use "@fremtind/jkl-feedback/feedback";
++ @use "@fremtind/jokul/styles/components/feedback";
+```
```

Comment:
Kanskje vi bør lenke til segmentet om stilark fra README her, siden mange team uansett importerer stilene for alle komponentene?

```diff
+ Les mer om hvordan du bruker stilark i `@fremtind/jokul` i [README-filen](./README.md#stilark)
```

Reply:
👍🏽 

Måtte legge det inn manuelt av en eller annen grunn, det kom ikke opp som en suggesten change.